### DISCUSSION TITLE
How to format date on header/footer

### DISCUSSION THREAD

**Discussion by pitermcflebor** [l1xnan/obsidian-better-export-pdf#358] [Q&A]
Hello,
I want to show the current year on the heading but adding `<span class="date"></span>` doesn't seem to allow formatting the date.

An easy way I suggest would be:
`<span class="date">yyyy</span>`
adding the format inside the span

**Comment by l1xnan** [comment]
This format is controlled by the underlying PDF engine and can not be modified. However, there are indeed solutions to address it at the upper level. I'll record this feature. Thank you for your feedback.

**Reply by DiamondStalker** [reply]
Regarding this functionality, is there already a way to change the format of the dates ?
